metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -187,8 +187,8 @@ def _create_lineage_map(self, entries: Iterable[QueryEvent]) -> Dict[str, Set[st
             ):
                 continue
             for ref_table in e.referencedTables:
-                destination_table_str = str(e.destinationTable)
-                ref_table_str = str(ref_table)
+                destination_table_str = str(e.destinationTable.remove_extras())
```

Comment:
Should this be the default behavior? Or behind a configuration flag?

Reply:
I can't think of a reason someone would want to have partition table names ingested- from all the folks we've spoken to this has appeared to be a bug!

Reply:
it is consistent to do this automatically by default since the rest of the connector does it. We can add configurability if needed later in the central function (remove_extras)